### PR TITLE
Update changelog and upgrade guide for 5.0.0

### PR DESCRIPTION
**Pull request by ralphtheninja**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,20 @@
...
+
+### Changed
+* Upgrade `abstract-leveldown` dependency from `~5.0.0` to `~6.0.0` (@ralphtheninja)
+* Upgrade `standard` devDependency from `^11.0.0` to `^12.0.0` (@ralphtheninja)
+* Tweak copyright years for less maintenance (@ralphtheninja)
+
+### Added
+* Add `nyc` and `coveralls` (@ralphtheninja)
+
+### Remove
```
Comment (vweevers) [inline CHANGELOG.md]:
```suggestion
 ### Removed
```

File: UPGRADING.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 This document describes breaking changes and how to upgrade. For a complete list of changes including minor and patch releases, please refer to the [changelog](CHANGELOG.md).
 
+## v5
+
+Upgraded `abstract-leveldown` to `v6.0.0`. Please see corresponding [changelog entry](https://github.com/Level/abstract-leveldown/blob/master/CHANGELOG.md#600---2018-10-20) for more information.
```
Comment (vweevers) [inline UPGRADING.md]:
```suggestion
 Upgraded `abstract-leveldown` to `v6.0.0`. Please see the corresponding [changelog entry](https://github.com/Level/abstract-leveldown/blob/master/CHANGELOG.md#600---2018-10-20) for more information.
```

And to make this referral less of a rabbit hole, I think we should reorder the `abstract-leveldown` changelog a bit. First describe changes in the public API (serialization etc, which is inherited by every implementation), then details which only matter to implementors (tests etc). I'll open an issue.

File: CHANGELOG.md (hunk 1 above)
Comment (ralphtheninja) [reply CHANGELOG.md]: I'm getting rusty.

**Comment by ralphtheninja** [comment]
Good to squash?

**Pull request merged**